content/docs/cucumber/configuration.md

```diff
@@ -15,36 +15,136 @@ polyglot:
 # Type Registry
 
 {{% block "java,kotlin" %}}
-Parameter types let you convert parameters from cucumber-expressions to objects. Data table and doc string string
-types let you convert data tables and doc strings to objects. Like step definitions type definitions are part of the
-glue. When placed on the glue path Cucumber will detect them automatically.
-
-For instance, the following class registers a `ParameterType` of type Color, and a `DataTableType` of type Person:
+Parameter types let you convert parameters from cucumber-expressions to objects. Data table and doc string string types let you convert data tables and doc strings to objects. Like step definitions, type definitions are part of the glue. When placed on the glue path Cucumber will detect them automatically.
+For example, the following class registers custom "Author" data table type, "List<Book>" docstring type and "Book" parameter type:
 
 ```java
-package com.example;
+package com.example.app;
 
 import io.cucumber.java.DataTableType;
+import io.cucumber.java.DocStringType;
 import io.cucumber.java.ParameterType;
+import io.cucumber.java.en.Given;
 
+import java.util.List;
 import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Steps {
```

Comment:
To make the distinction between steps in gherkin and the step _definitions_ in java clear we should name this class  `StepDefinitions`.
```suggestion
public class StepDefinitions {
```